### ISSUE TITLE
Error when running java –jar octane-gitlab-service-<version>.jar

### ISSUE THREAD

**Issue by fatimalaoui** [MicroFocus/octane-gitlab-service#3]
Hi all, i'm trying to connect uft to octane using gitlab.
I have errors after running this command : java –jar octane-gitlab-service-<version>.jar
I want to make sure that the gitlab-runner file and application-properties files are OK.
Can someone help me to verify them please?
Thank you in advance
[config.txt](https://github.com/MicroFocus/octane-gitlab-service/files/3141122/config.txt)
[application.properties.txt](https://github.com/MicroFocus/octane-gitlab-service/files/3141128/application.properties.txt)

**Comment by shawns1** [comment]
This is not correct:  gitlab.personalAccessToken=token //**token i took from my gitlab project>Setting>CI>runner..**

Runner tokens are used to connect a runner to a specific project.  Personal access tokens are generated by the user to authenticate to the api service on behalf of that user.

The token is the personal access token for your account in the user settings located here:
http(s)://{my.gitlabserver.com}/profile/personal_access_tokens
